packages/ember/src/components/footer.gts

```diff
@@ -0,0 +1,31 @@
+import type { TOC } from '@ember/component/template-only';
+
+interface FooterSignature {
+  Element: HTMLElement;
+  Blocks: {
+    left?: [];
+    right?: [];
+  };
+}
+
+const Footer: TOC<FooterSignature> = <template>
+  <footer
+    class="bg-body-secondary border-top border-2 border-dark-subtle mt-auto fs-6"
+    ...attributes
+  >
+    <div class="container-fluid py-2 px-4">
```

Comment:
```suggestion
    <div class="container-fluid py-2 px-2 px-sm-4 ">
```

This will keep turn current padding except at the smallest breakpoint and reduce the horizontal padding to try to save as much space for content as we can.